app/src/main/java/campus/tech/kakao/map/onItemSelected.kt

```diff
@@ -0,0 +1,6 @@
+package campus.tech.kakao.map
+
+interface onItemSelected {
```

Comment:
사소한 것이지만, interface 또한 대문자로 시작하면 좋을 것 같고, listener 라면 postfix 로 ####Listener 로 붙여주시면 조금 더 보기 좋습니다!
```suggestion
interface OnItemSelectedListener
```

Reply:
우와 빠른 리뷰 감사합니다~!